www/js/settings.js

```diff
@@ -5,6 +5,29 @@ var setting_lastj = -1
 var current_setting_filter = 'nvs'
 var setup_is_done = false
 var do_not_build_settings = false
+const CONFIG_TOOLTIPS = {
+  Maslow_vertical: "If the maslow is oriented horizontally, set this to false",
+  Maslow_calibration_offset_X: "mm offset from the edge of the frame, Y",
+  Maslow_calibration_offset_Y: "mm offset from the edge of the frame, Y",
+  Maslow_calibration_size_X: "Number of X points to use in calibration",
+  Maslow_calibration_size_Y: "Number of Y points to use in calibration",
+  Maslow_brX: "Bottom right anchor x (normally width in mm)",
+  Maslow_brY: "Bottom right anchor y (normally 0)",
+  Maslow_brZ: "Bottom right z (normally 117)",
+  Maslow_tlX: "Top left anchor x (normally 0)",
+  Maslow_tlY: "Top left anchor y (normally height in mm)",
+  Maslow_tlZ: "Top left z (normally 144)",
+  Maslow_trX: "Top right anchor x (normally width in mm)",
+  Maslow_trY: "Top right anchor y (normally height in mm)",
+  Maslow_trZ: "Top right z (normally 97)",
+  Maslow_blX: "Bottom left anchor x (normally 0)",
+  Maslow_blY: "Bottom left anchor y (normally 0)",
+  Maslow_blZ: "Bottom left z (normally 75)",
+  Maslow_Retract_Current_Threshold: "Sets how hard should Maslow pull on the belts to retract before considering them to be all the way in",
+  Maslow_Calibration_Current_Threshold: "Sets how hard should Maslow pull on the belts during the calibration process.",
+  Maslow_calibration_extend_top_y: "starting Y for top belts on extend all (-1000 to 1000) default 0",
+  Maslow_calibration_extend_bottom_y: "starting Y for bottom belts on extend all (-1000 to 1000) default ",
+}
```

Comment:
These last two are something I'm working on in fluidnc that I will PR another day for the extra belt extend we talked about

Reply:
Brilliant!